Toolchain/Patches/llvm/0005-libc-Add-support-for-SerenityOS.patch

```diff
@@ -109,20 +102,6 @@ index 000000000..0f939d2f6
 +#endif // __serenity__
 +
 +#endif
-diff --git a/libcxx/include/initializer_list b/libcxx/include/initializer_list
-index fefaf8cf8..c388bc246 100644
---- a/libcxx/include/initializer_list
-+++ b/libcxx/include/initializer_list
-@@ -43,7 +43,9 @@ template<class E> const E* end(initializer_list<E> il) noexcept; // constexpr in
- */
- 
- #include <__config>
-+#if !defined(__serenity__) || !defined(KERNEL)
```

Comment:
@timschumi was this required for serenity-on-serenity? It built everything fine in a cross-compile env.

Reply:
I don't recall having to change that, so probably not?

Reply:
🤷‍♂️ cool, so one less patch to carry then. I'm guessing this was part of the initial clang 13 patches and we've since fixed the header issue it was papering over.